src/TesApi.Web/Program.cs

```diff
@@ -35,11 +35,12 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
             Console.WriteLine($"TES v{Startup.TesVersion} build {System.Reflection.Assembly.GetExecutingAssembly().GetName().Version}");
 
             Options.ApplicationInsightsOptions applicationInsightsOptions = default;
-            var builder = WebHost.CreateDefaultBuilder<Startup>(args);
+            var builder = WebHost.CreateDefaultBuilder<Startup>(args)
+                .UseUrls("http://0.0.0.0:80");
```

Comment:
All the examples I see never have explicit settings mixed with configuration. The configuration is one step removed from the settings and it's possible that explicit settings will cause configurations to be ignored. I'd feel safer if this was implemented like this:

```
            if (string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("ASPNETCORE_URLS")))
            {
                builder.UseUrls("http://0.0.0.0:80");
            }
```

Reply:
Or, put this in appsettings.json instead of c#